### PR TITLE
cache l10n instance in verifyPath

### PR DESCRIPTION
**Pull request by icewind1991**
More preventing expensive container queries

[comparison](https://blackfire.io/profiles/compare/338c78dd-e804-468c-963f-380462555ab8/graph)

cc @DeepDiver1975 @rullzer

### CODE REVIEW

**Comment by rullzer** [comment]
Same here. ;)
:+1:

File: lib/private/files/view.php [hunk 1]
```diff
@@ -1723,8 +1723,10 @@ private function getPartFileInfo($path) {
 	 * @throws InvalidPathException
 	 */
 	public function verifyPath($path, $fileName) {
-
-		$l10n = \OC::$server->getL10N('lib');
```
Comment (DeepDiver1975) [inline lib/private/files/view.php]: hmmm - the server container should cache all instances which are created - optimizations like this should not be necessary ...

**Comment by MorrisJobke** [comment]
:warning: May be superseded by https://github.com/owncloud/core/pull/21419

File: lib/private/files/view.php [hunk 2]
```diff
@@ -1723,8 +1723,10 @@ private function getPartFileInfo($path) {
...
+		static $l10n;
```
Comment (nickvergessen) [inline lib/private/files/view.php]: if you cache it locally, use a member not static, thanks

**Comment by DeepDiver1975** [comment]
@icewind1991 we will try to patch pimple to get more performance out of the container.

@rullzer volunteered to have a look at pimple.

Do we agree to close this PR?

**Comment by MorrisJobke** [comment]
> Do we agree to close this PR?

Yes.

**Pull request closed without merging**